cmd/commands/runtime_install.go

```diff
@@ -196,8 +195,11 @@ func NewRuntimeInstallCommand() *cobra.Command {
 
 			createAnalyticsReporter(ctx, reporter.InstallFlow, installationOpts.DisableTelemetry)
 
-			if (accessMode != "") {
+			if accessMode != "" {
 				installationOpts.AccessMode = platmodel.AccessMode(strings.ToUpper(accessMode))
+				if installationOpts.AccessMode == platmodel.AccessModeTunnel && installationOpts.IngressHost != "" {
+					return fmt.Errorf("ingress host can't be set when access mode is Tunnel.")
+				}
```

Comment:
maybe this should be after the next condition? make sure accessMode is valid first?

Reply:
done